frontend/src/containers/ResetPassword/ResetPasswordPage.js

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from 'react';
+import LogInPageHeader from '../../components/LogInPageHeader/LogInPageHeader';
+import NewPassword from '../../components/NewPassword/NewPassword';
+import Footer from '../../components/Footer/Footer';
+import './ResetPasswordPage.css';
+
+class ResetPasswordPage extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      userId: '', // eslint-disable-line react/no-unused-state
+    };
+  }
+
+  changeAuthorizedState = (returnUserId) => {
```

Comment:
I think we can just remove this, since we're not presently using it and it likely won't be useful later.